### PR TITLE
no need to write timestamp and nonce in op_return

### PR DESCRIPTION
**Pull request by BenedictKing**
This significantly reduces the fee rate of the reveal transaction compared to the expected amount.

### CODE REVIEW

**Comment by lucky2077** [comment]
Without the op_return data, can we genrate a revealtx match mint_bitworkr prefix?

**Comment by BenedictKing** [comment]
> Without the op_return data, can we genrate a revealtx match mint_bitworkr prefix?

of course, a revealtx matched the mint_bitworkr prefix is needed. but we don't have to write the calculation in op_return, it is a huge waste.

File: lib/utils/atomical-operation-builder.ts [hunk 1]
```diff
@@ -937,15 +937,7 @@ export class AtomicalOperationBuilder {
...
             }
-            const data = Buffer.from(unixTime + ":" + nonce, "utf8");
-            const embed = bitcoin.payments.embed({ data: [data] });
 
-            if (performBitworkForRevealTx) {
-                psbt.addOutput({
-                    script: embed.output!,
-                    value: 0,
-                });
-            }
```
Comment (atomicals) [inline lib/utils/atomical-operation-builder.ts:948]: How will the proof of work function?

Comment (BenedictKing) [reply lib/utils/atomical-operation-builder.ts:948]: how about do it like commit tx using different sequence?

Comment (BenedictKing) [reply lib/utils/atomical-operation-builder.ts:948]: maybe we can increase fee of the reveal tx

**Pull request closed without merging**